### PR TITLE
General repository cleanup 2

### PR DESCRIPTION
**Pull request by SepandKashani**
Follow-up fixes to https://github.com/imagingofthings/pyFFS/pull/12.

### CODE REVIEW

File: doc/general/install.rst [hunk 1]
```diff
@@ -10,29 +10,24 @@ Installation
...
 
-* pyFFS is developed and tested on x86_64 systems running Linux.
-
-* It is recommended to install dependencies using `Miniconda <https://conda.io/miniconda.html>`_ or
-  `Anaconda <https://www.anaconda.com/download/#linux>`_::
-
-    $ conda install --channel=defaults    \
-                    --channel=conda-forge \
-                    --file=requirements.txt
+pyFFS is developed and tested on x86_64 systems running Linux.
```
Comment (ebezzam) [inline doc/general/install.rst:33]: could also add macOS Catalina here

**Review by ebezzam — COMMENTED** [review]
Looks good

File: doc/index.rst [hunk 2]
```diff
@@ -12,17 +14,44 @@ pyFFS documentation
 *pyFFS* is a library to compute Fourier Series.
```
Comment (ebezzam) [inline doc/index.rst:14]: do you think we should make this more specific? something like:

> pyFFS is a library to efficiently compute and interpolate Fourier Series coefficients.

Comment (SepandKashani) [reply doc/index.rst:14]: The main goal is to bring FS-based tools to the general public, so I think
```
pyFFS is a collection of efficient algorithms to compute Fourier Series and related transforms.
```
fits the bill.

Comment (SepandKashani) [reply doc/index.rst:14]: What do you think? In any case we should elaborate on the contents. The initial line is too sparse.

**Pull request merged**